__docs__/wonder-blocks-form/text-field.stories.tsx

```diff
@@ -137,21 +138,46 @@ export const Number: StoryComponentType = () => {
     };
 
     return (
-        <TextField
-            id="tf-3"
-            type="number"
-            value={value}
-            placeholder="Number"
-            onChange={handleChange}
-            onKeyDown={handleKeyDown}
-        />
+        <View style={styles.column}>
+            <TextField
+                id="tf-3"
+                type="number"
+                value={value}
+                placeholder="Number"
+                onChange={handleChange}
+                onKeyDown={handleKeyDown}
+            />
+            <Strut size={spacing.small_12} />
+            <Body>
+                The following text field has a min of 0, a max of 15, and a snap
```

Comment:
"snap" or "step"?  The prop is named `step` but is "snap" another term for it?

Reply:
You're right! It's just step.